register.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
```

Comment:
This file should probably be at `src/register.js` to match the rest of the project structure. I know it doesn't make the `require` call quite as nice though.

Reply:
If I am not mistaken, `node -r polyfill-package/register target.js` is convention, eg [node-window-polyfill](https://www.npmjs.com/package/node-window-polyfill), [any-observable](https://www.npmjs.com/package/any-observable) and others. Usually it is put in `src/registed` in case if that is built and published to npm as `pkgname/register`. Because `browser-env` does not have bundling, I guess it is just easier to keep direct `register` entry.

Reply:
Ok, I'm fine with that.